www.foia.gov/_sass/modules/_three_column_summary.scss

```diff
@@ -0,0 +1,48 @@
+.three-column-summary {
+  margin: $space-4x auto;
+  .secondary-text:after {
+    content: "";
+    @include text-divider;
+  }
+  .usa-width-one-third {
+    p {
+      @media screen and (max-width: $medium-screen) {
+        margin-left: 50px; // the width of the bubble at this screen size + margin
```

Comment:
Probably should be a variable, not for re-use but so it carries a name.

Reply:
I have never used variables like that and I feel a little silly for not realizing how useful that can be!